tests/lib/shared/config-validator.js

```diff
@@ -244,7 +244,7 @@ describe("Validator", () => {
             it("should throw with an object", () => {
                 const fn = validator.validate.bind(null, { extends: {} }, null, ruleMapper);
 
-                assert.throws(fn, "Property \"extends\" is the wrong type (expected string/array but got `{}`).");
+                assert.throws(fn, "ESLint configuration in null is invalid:\n\t- Property \"extends\" is the wrong type (expected string but got `{}`).\n\t- Property \"extends\" is the wrong type (expected array but got `{}`).\n\t- \"extends\" should match exactly one schema in oneOf. Value: {}.");
```

Comment:
the left-side message seems more readable to me. :)

Reply:
I agree. But this message depends on JSON Scheme and Ajv package...

Reply:
I see. Hopefully we can improve it by using something like custom error messages. but it don't have to be done in this PR. 😄